.github/workflows/cpp-packaging.yml

```diff
@@ -433,7 +433,7 @@ jobs:
           for pkg in artifacts/firebase-cpp-sdk-${{ matrix.sdk_platform }}${{ matrix.suffix }}*-build/*.tgz; do
             # determine the build variant based on the artifact filename
```

Comment:
Just a general comment on this step and not this particular line. 
Is this step supposed to run on all runners (including Windows)?

Reply:
It runs on Linux runners for packaging the Linux and Windows SDKs.
It runs on MacOS runners for packaging the Mac SDK (as it needs to use mac-specific tools like lipo and libtool).